src/Rings/localization_interface.jl

```diff
@@ -175,14 +176,9 @@ function parent(f::AbsLocalizedRingElem)
   error("`parent` is not implemented for the type $(typeof(f))")
 end
 
-### default functionality for printing
-function Base.show(io::IO, f::AbsLocalizedRingElem)
-  if needs_parentheses(f)
-    print(io, "($(numerator(f)))//($(denominator(f)))")
-  else 
-    print(io, "$(numerator(f))//$(denominator(f))")
-  end
-end
+expressify(f::AbsLocalizedRingElem; context=nothing) = Expr(:call, ://, expressify(numerator(f)), expressify(denominator(f)))
```

Comment:
```suggestion
expressify(f::AbsLocalizedRingElem; context=nothing) = Expr(:call, ://, expressify(numerator(f), context = context), expressify(denominator(f), context = context))
```

Reply:
I see. That makes sense. Just in general: Do you wish for me to go through the code and find other instances, where expressify should be used? If yes: What are the criteria for that? I.e. in which cases do you wish for expressify to replace the other show routines? Maybe we could meet and talk about this in person to avoid lengthy iterations of such small snippets as the above one. Let me know (e.g. on slack).

Reply:
Unless someone complains about ugly printing (e.g. ugly printing for polynomials over some new ring), I would not worry about changing it personally. Here we just wanted to remove the call to the soon to be retired functions.